app/controllers/UpdateRelationshipController.scala

```diff
@@ -80,24 +79,23 @@ class   UpdateRelationshipController @Inject()(
         case NonFatal(_) => Ok(decisionV(CheckClaimOrCancelDecisionForm.form))
       }
   }
-
+//TODO - Refactor to address warning
```

Comment:
Has this been done?

Reply:
This was added following a discussion with Kurt, will likely result in a separate ticket, I'll raise further discussion with him.